components/datatypes/industry-verticals/claim.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/claim",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Claim",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Captures fields related to a claim when initated by a customer.",
+  "definitions": {
+    "claim": {
+      "properties": {
+        "xdm:ID": {
+          "title": "Claim ID",
+          "type": "string",
+          "description": "Unique identifier for the claim."
+        },
+        "xdm:startDate": {
+          "title": "Start Date",
+          "type": "string",
+          "format": "date-time",
+          "description": "Date the claim was successfully submitted."
+        },
+        "xdm:status": {
+          "title": "Status",
+          "type": "string",
+          "enum": ["initiated", "in progress", "approved", "declined"],
+          "description": "Captures the status of the claim for any given point in the application process."
+        },
+        "xdm:policyID": {
+          "title": "Policy ID",
+          "type": "string",
+          "description": "Policy Identifier."
+        },
+        "xdm:benefitAmount": {
+          "title": "Benefit Amount",
+          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "description": "Captures the benefit amount."
+        },
+        "xdm:steps": {
```

Comment:
Please remove steps as we previously discussed. We can add this later if required

Reply:
There are still instances - particularly with Analytics - where we will use this.  Can we leave it in as deprecated for now?

Reply:
We should only deprecate things that once existed as stable, not introduce them before they ever existed as deprecated. Is it ok if we remove this for now, @jbranson000 ?

Reply:
once we remove that we can push this PR through :)

Reply:
Of course - that'll be fine.  I went ahead and removed it.